client/src/pages/myProfile.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import {getPortfolioItems} from "../services/portfolioService.js"
+import Portfolio from "../components/Portfolio"
+import { Doughnut } from 'react-chartjs-2'
+import {generateRandomColor, createChartInput} from '../utils'
+
+const piestate = {
+  labels: ['January', 'February', 'March',
+                  'April', 'May'],
+          datasets: [
+          {
+              label: 'Rainfall',
+              backgroundColor: [
+              '#B21F00',
+              '#C9DE00',
+              '#2FDE00',
+              '#00A6B4',
+              '#6800B4'
+              ],
+              hoverBackgroundColor: [
+              '#501800',
+              '#4B5000',
+              '#175000',
+              '#003350',
+              '#35014F'
+              ],
+              data: [65, 59, 80, 81, 56]
+          }
+          ]
+};
+
+class Profile extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            isFetching: false,
+            items: [],
+            chartstate: {
+              labels: [],
+              datasets: []
+            },
+        };
+    }
+    async fetchItems() {
+        try {
+            this.setState({...this.state, isFetching: true});
+            const response = await getPortfolioItems();
+            this.setState({items: response.data.portfolioItems, isFetching: false});
+        } catch (e) {
+            console.log(e);
+            this.setState({...this.state, isFetching: false});
+        }
+    };
+    
+    componentDidMount() {
+      this.fetchItems();     
+    }
+
+    render() {
+      const {items} = this.state;
+      const chartinput = createChartInput(items);
+      console.log("chart input below");
+      console.log(chartinput);
+      console.log("piestate below");
+      console.log(piestate);
```

Comment:
Can we remove the console logs before merging to master? 🤣

Reply:
oh haha, sorry about that! I forgot to remove them after testing! I'll fix it right now